### ISSUE TITLE
After updating to Preview 14, SkiaSharp throws: Type SkiaSharp.Views.Maui.Handlers.SKCanvasViewHandler has invalid vtable method slot 39 with method none

### ISSUE THREAD

**Issue by SailDev** [dotnet/maui#5371]
### Description

In my app i have a large and complex custom control, built with SkiaSharp.

Maui Preview 13 worked fine, but after upgrading to Preview 14, the following runtime exception is thrown:

System.TypeLoadException
  Message=Type SkiaSharp.Views.Maui.Handlers.SKCanvasViewHandler has invalid vtable method slot 39 with method none

PLEASE NOTE:
Version with bug is Preview 14.
At this point in time, the github form did not offer this option.

### Steps to Reproduce

Create a custom control with SkiaSharp.

### Version with bug

Unknown/Other (please specify)

### Last version that worked well

Preview 13

### Affected platforms

Android

### Affected platform versions

Physical device with Android 11

### Relevant log output

```shell
System.TypeLoadException
Message=Type SkiaSharp.Views.Maui.Handlers.SKCanvasViewHandler has invalid vtable method slot 39 with method none
```

**Comment by jfversluis** [comment]
@mattleibow ?

**Comment by SailDev** [comment]
Cleaned and rebuilt everything.

Forgot to mention it, i use the latest preview version of SkiaSharp, offered by VS nuget manager:

`
<PackageReference Include="SkiaSharp.Views.Maui.Controls" Version="2.88.0-preview.209" />
<PackageReference Include="SkiaSharp.Views.Maui.Controls.Compatibility" Version="2.88.0-preview.209" />
<PackageReference Include="SkiaSharp.Views.Maui.Core" Version="2.88.0-preview.209" />
`

**Comment by mattleibow** [comment]
@SailDev seems the SkiaSharp release had some delay on indexing or something. There should be a preview 232 out now.

**Comment by SailDev** [comment]
Updated to .232. Tested. Works!
MANY thanks to everyone for helping.

**Comment by SailDev** [comment]
Fixed by SkiaSharp version 2.88.0-preview.232.
